src/ci/resources/tes_application.conf.ctmpl

```diff
@@ -0,0 +1,81 @@
+include required(classpath("application.conf"))
+include "build_application.inc.conf"
+
+filesystems {
```

Comment:
If people have opinions, I'm happy to update comments, remove unused keys, or change defaults.